src/main/java/org/jabref/gui/icon/IconTheme.java

```diff
@@ -2,64 +2,82 @@
+import org.kordamp.ikonli.Ikon;
+import org.kordamp.ikonli.IkonProvider;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignA;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignB;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignC;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignD;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignE;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignF;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignG;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignH;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignI;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignK;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignL;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignM;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignN;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignO;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignP;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignR;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignS;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignT;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignU;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignV;
+import org.kordamp.ikonli.materialdesign2.MaterialDesignW;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static java.util.EnumSet.allOf;
+
 public class IconTheme {
 
     public static final Color DEFAULT_DISABLED_COLOR = Color.web("#c8c8c8");
     public static final Color SELECTED_COLOR = Color.web("#50618F");
     private static final String DEFAULT_ICON_PATH = "/images/external/red.png";
     private static final Logger LOGGER = LoggerFactory.getLogger(IconTheme.class);
     private static final Map<String, String> KEY_TO_ICON = readIconThemeFile(IconTheme.class.getResource("/images/Icons.properties"), "/images/external/");
+    private static final Set<Ikon> ICON_NAMES = new HashSet<>();
 
-    public static void loadFonts() {
-        try (InputStream stream = getMaterialDesignIconsStream()) {
-            Font.loadFont(stream, 7);
-        } catch (IOException e) {
-            LOGGER.error("Error loading Material Design Icons TTF font", e);
-        }
-
-        try (InputStream stream = getJabRefMaterialDesignIconsStream()) {
-            Font.loadFont(stream, 7);
-        } catch (IOException e) {
-            LOGGER.error("Error loading custom font for custom JabRef icons", e);
-        }
+    public static Color getDefaultGroupColor() {
+        return Color.web("#8a8a8a");
     }
 
-    private static InputStream getMaterialDesignIconsStream() {
-        return IconTheme.class.getResourceAsStream("/fonts/materialdesignicons-webfont.ttf");
+    public static Optional<JabRefIcon> findIcon(String code, Color color) {
+        return ICON_NAMES.stream().filter(icon -> icon.toString().equals(code.toUpperCase(Locale.ENGLISH)))
```

Comment:
Maybe make `ICON_NAMES` a hashmap? Looking up the icon via the filter can otherwise become quite expensive.

Reply:
And maybe initalize `ICON_NAMES` lazily, i.e. only when `findIcon` is called.